guidelines/pagination.md

```diff
@@ -0,0 +1,60 @@
+# Pagination
+
+Pagination is a feature introduced into the API to manage requests for large amounts of data.
+It enables us to deliver data in a measured way that does not put a strain on performance for all API users.
+We anticipate rolling out new versions of popular 'Get' operations which will incorporate pagination.

```

Comment:
I'm not sure that `anticipate` is the right word since we are the ones in charge. Maybe something like `This concept is one of ways how the API would be improved in future`? Or omit it all together?

Reply:
Ok, fair enough, I have re-written the sentence to: "We are rolling out new versions of popular 'Get' operations to incorporate pagination, e.g. [Get all reservations](../operations/reservations.md#get-all-reservations)."